roles/kubernetes/node/tasks/kubelet.yml

```diff
@@ -18,6 +18,35 @@
     - kubelet
     - kubeadm
 
+- name: Gather facts about the node
```

Comment:
Might want to add a when `kube_enable_auto_reserved_resources` here.

Reply:
added it. Thank you for your review.